src/hexen/h2_main.c

```diff
@@ -357,6 +358,81 @@ void D_SetGameDescription(void)
     }
 }
 
+static struct
+{
+    const char *description;
+    const char *cmdline;
+    GameVersion_t version;
+} gameversions[] = {
+    {"Hexen 1.1",            "1.1",        exe_hexen_1_1},
+    {"Hexen 1.1 (alt)",      "1.1r2",      exe_hexen_1_1r2},
+    { NULL,                  NULL,         0},
+};
+
+// Initialize the game version
+
+static void InitGameVersion(void)
+{
+    int p;
+    int i;
```

Comment:
The scope of `i` could be reduced.

Reply:
What's been written by me about the const qualifier also holds here, albeit just to Strife (and Hexen), not Doom.